### PR TITLE
Add support for Python 3 in Android and iOS

### PR DESCRIPTION
**Pull request by dharmeshmp**
#### Changes

- Added python 3 command support to Android and iOS

#### Checklist

- [x] Read the contribution [guide](../CONTRIBUTING.md) and accept the [code](../CODE_OF_CONDUCT.md) of conduct
- [x] Readme (updated or not needed)

### CODE REVIEW

**Comment by webbertakken** [comment]
When you say python2, do you mean python3?

**Comment by dharmeshmp** [comment]
I have implemented second suggestion.
https://github.com/game-ci/docker/issues/100#issuecomment-1023476647

**Comment by webbertakken** [comment]
> I have implemented second suggestion. [#100 (comment)](https://github.com/game-ci/docker/issues/100#issuecomment-1023476647)

Much appreciated!

Would you mind updating the PR title and comments in the code to reflect the correct version please?

**Comment by dharmeshmp** [comment]
of course you can do any thing.

File: images/ubuntu/editor/Dockerfile [hunk 1]
```diff
@@ -205,6 +205,14 @@ RUN echo "$module" | grep -q -v '\(webgl\|linux-il2cpp\)' \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/*
 
+#=======================================================================================
+# [android, ios] python 2 command support
```
Comment (webbertakken) [inline images/ubuntu/editor/Dockerfile:209]:
```suggestion
# [android, ios] python 3 support
```

**Review by webbertakken — CHANGES_REQUESTED** [review]

**Comment by webbertakken** [comment]
> of course you can do any thing.

Done.

Did you build the image and check if it solves your problem?

**Comment by dharmeshmp** [comment]
> Did you build the image and check if it solves your problem?

No, But I checked by adding `ln -s /usr/bin/python3 /usr/bin/python` in before_script stage of gitlab.

**Comment by webbertakken** [comment]
> > Did you build the image and check if it solves your problem?
> 
> No, But I checked by adding `ln -s /usr/bin/python3 /usr/bin/python` in before_script stage of gitlab.

Ok good enough. Can you confirm that that worked?

**Comment by dharmeshmp** [comment]
> Ok good enough. Can you confirm that that worked?

Yes, It's working fine for me.

**Comment by mastef** [comment]
Hi, @dharmeshmp @webbertakken 

this PR was based on my comment. After further review I would have recommended to just run the previous step for all targets :
image

```diff
-RUN echo "$module" | grep -q -v '\(webgl\|linux-il2cpp\)' \
-  && exit 0 \
-  || : \
-  && apt-get -q update \
+RUN apt-get -q update \
  && apt-get -q install -y --no-install-recommends --allow-downgrades \
    python \
    python-setuptools \
    build-essential \
    clang \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
```
As that already installs `python`. It would also remove additional complexity.

**Comment by mastef** [comment]
Or something like 

```diff
+RUN apt-get -q install -y python
 RUN echo "$module" | grep -q -v '\(webgl\|linux-il2cpp\)' \
   && exit 0 \
   || : \
   && apt-get -q update \
   && apt-get -q install -y --no-install-recommends --allow-downgrades \
-    python \
     python-setuptools \
     build-essential \
     clang \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/*
```

**Comment by mastef** [comment]
( There's nothing wrong with the current implementation, I just noticed that python is being installed in a previous step. And for some reason I need it now also for `win/mac/unix` installs as Firebase is not filtering them in the latest version. So having `python` in all containers may be the fastest fix )

**Pull request merged**